services/user/Tokens/ui/package.json

```diff
@@ -19,33 +21,40 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-tabs": "^1.0.4",
+    "@react-three/fiber": "^8.17.10",
     "@tanstack/react-query": "^5.32.1",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
     "framer-motion": "^11.2.6",
+    "konva": "^9.3.16",
     "lucide-react": "^0.373.0",
     "next-themes": "^0.3.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.51.3",
+    "react-konva": "^18.2.10",
+    "react-native": "^0.76.2",
     "react-spring": "^9.7.3",
+    "react-zdog": "^1.2.2",
     "sonner": "^1.4.41",
     "tailwind-merge": "^2.3.0",
     "tailwindcss-animate": "^1.0.7",
+    "three": "^0.170.0",
     "usehooks-ts": "^3.1.0",
     "vaul": "^0.9.0",
+    "zdog": "^1.1.3",
```

Comment:
I'm seeing new packages introduced here too. What's that about?

Reply:
`react-spring` is an umbrella library for multiple platforms. It would be better here to remove these and change the `react-spring` library to `@react-spring/web`. That will make the warnings disappear. The import in `AnimateNumber.tsx` will also need to be updated.

Would you like me to push a commit to this branch with those changes?